core-application/src/test/java/gohigher/application/port/in/ApplicationProcessQueryServiceTest.java

```diff
@@ -0,0 +1,62 @@
+package gohigher.application.port.in;
+
+import static gohigher.application.ApplicationErrorCode.*;
+import static gohigher.common.ProcessType.*;
+import static org.assertj.core.api.Assertions.*;
+import static org.mockito.Mockito.*;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import gohigher.application.port.out.persistence.ApplicationPersistenceQueryPort;
+import gohigher.application.port.out.persistence.ApplicationProcessPersistenceQueryPort;
+
+@DisplayName("ApplicationProcessQueryService 클래스의")
+@ExtendWith(MockitoExtension.class)
+class ApplicationProcessQueryServiceTest {
+
+	@Mock
+	private ApplicationPersistenceQueryPort applicationPersistenceQueryPort;
+	@Mock
+	private ApplicationProcessPersistenceQueryPort applicationProcessPersistenceQueryPort;
+	@InjectMocks
+	private ApplicationProcessQueryService applicationProcessQueryService;
+
+	@DisplayName("findByApplicationIdAndProcessType 메서드는")
+	@Nested
+	class Describe_FindByApplicationIdAndProcessType {
+
+		@DisplayName("id와 userId가 일치하는 지원서가 없는 경우에, ")
```

Comment:
```suggestion
		@DisplayName("id와 userId가 일치하는 지원서가 없는 경우에")
```
DisplayName 에 따라 한 줄씩 작성되므로 콤마나 공백은 없어도 될것 같습니다!